answer/166ryoryo/JS/05/randomBgColor.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="ja">
+<head>
+<meta charset="utf-8">
+<title></title>
+<meta name="keywords" content="">
+<meta name="description" content="">
+<script LANGUAGE="JavaScript">
+  function rangeRandom(num1,num2)	{
+    if((num1 - num2) > 0)	{
```

Comment:
```suggestion
    if((num1 - num2) > 0) {
```

tabではなくスペース使うようにしましょう！